.asf.yaml

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+github:
+  enabled_merge_buttons:
+    squash: true
+    merge: false
+    rebase: false
+notifications:
+  commits:      common-commits@hadoop.apache.org
+  issues:       common-issues@hadoop.apache.org
+  pullrequests: common-issues@hadoop.apache.org
+  jira_options: link label worklog
```

Comment:
nit: do we need newline at the end of file?

Reply:
The link was added by me only. But I got this checked from the Infra team on the asfInfra slack channel and they only told, post this things should work.

Regarding the newline, I don't know if it is required, Most of the files don't have it. If you say I will push a commit adding one

Reply:
No it's not required. It's not a blank line newline, but a hidden char. Sharing some thoughts:

1. No end of line char can lead to unexpected behavior when using other tools on the file. For e.g. git diff or external diff tools depend on newlines to generate a correct diff. `wc` command counting lines may also be broken. More can be found at [this discussion](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795#729795).
2. Next time when someone edits the file, some editors will automatically add a "newline" char at the end of this file - for good intention. If so, seems like they "changed" the last line of this file in their PR which they do not.

But that is not a real problem here. We can commit without fixing that, I'm fine with that.